src/components/app.test.ui.tsx

```diff
@@ -87,24 +88,30 @@ describe('Voyager', () => {
   describe('vega-lite spec', () => {
     it('accepts valid spec', done => {
       const config = {};
-      const data: any = undefined;
       const store = configureStore();
 
+      const values = [
+        {date: "24-Apr-07", close: "93.24"},
+        {date: "25-Apr-07", close: "95.35"},
+        {date: "26-Apr-07", close: "98.84"},
+        {date: "27-Apr-07", close: "99.92"},
+      ];
+      const data: Data = {values};
       const spec: Object = {
         "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
         "data": {
           "values": [
-            {"date": "24-Apr-07", "close": "93.24"},
-            {"date": "25-Apr-07", "close": "95.35"},
-            {"date": "26-Apr-07", "close": "98.84"},
-            {"date": "27-Apr-07", "close": "99.92"}
+            {"date": "24-Apr-07", "binclose": "93.24"},
```

Comment:
Why changing this?   -- close is already right and the field is binned in the spec